src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,110 @@
+input {
+  padding: 22px 20px;
+}
+
+.signUp {
+  width: 400px;
+  margin: 0 auto;
+  padding-top: 80px;
+  .signUpTitle {
+    border-bottom: 1px solid lightgray;
+    font-size: 32px;
+    text-align: center;
+    font-weight: 700;
+    line-height: 42px;
+    padding-bottom: 40px;
+  }
+  .infTitle {
+    font-size: 20px;
+    font-weight: 700;
+    padding: 30px 0;
+  }
```

```suggestion
.signUp {
  width: 400px;
  margin: 0 auto;
  padding-top: 80px;
  
  .signUpTitle {
    border-bottom: 1px solid lightgray;
    font-size: 32px;
    text-align: center;
    font-weight: 700;
    line-height: 42px;
    padding-bottom: 40px;
  }
  
  .infTitle {
    font-size: 20px;
    font-weight: 700;
    padding: 30px 0;
  }
```
스타일 블록 끼리는 가독성을 위해 한칸씩 띄워주세요!